enhancements/cert-manager/istio-csr-controller.md

```diff
@@ -690,7 +963,10 @@ None
 
 ### Tech Preview -> GA
 
-N/A. This feature is for Tech Preview, until decided for GA.
+- Feature is enabled by default, with option to disable.
```

Comment:
Once GA'd, should we give the users the option to disable it? In general OpenShift features, there is no option go back once GA'd.